engine/src/main/java/org/terasology/world/chunks/Chunk.java

```diff
@@ -19,6 +19,7 @@
 import org.terasology.module.sandbox.API;
 
 /**
+ * This interface bundles together all the methods chunk contains.
```

Comment:
The name of the interface is Chunk, so obviously it will bundle the methods of chunk - doesn't really help the reader. I would like to see a more concrete description of this (explain what a chunk is).